### ISSUE TITLE
v1.12 AMI builds fail to respond to kubectl logs

### ISSUE THREAD

**Issue by willthames** [awslabs/amazon-eks-ami#281]
**What happened**:
kubectl logs against a node running an AMI built for kubernetes 1.12 doesn't work

**What you expected to happen**:

```
kubectl logs $pod
<some logs>
```

**What I actually get**:
```
Error from server: Get https://$node:10250/containerLogs/monitoring/$pod/$container: remote error: tls: handshake failure
```

**How to reproduce it (as minimally and precisely as possible)**:
Build a 1.12 EKS AMI, and run it in an EKS cluster

**Anything else we need to know?**:

This appears to be related to the 
```
  "tlsCipherSuites": ["TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"]
```
setting in kubelet-config-with-secret-polling.json

If I remove that setting, kubectl logs works.

I can also confirm that `openssl s_client -connect $node:443` suggests that the connection is unencrypted before I remove the `tlsCipherSuites` line, and is properly encrypted after removing the line. So somehow, setting tlsCipherSuites seems to disable TLS - which is completely unexpected, hence this issue!

**Environment**:
- AWS Region: ap-southeast-2
- Instance Type(s): c5.2xlarge
- EKS Platform version (use `aws eks describe-cluster --name <name> --query cluster.platformVersion`): eks-2
- Kubernetes version (use `aws eks describe-cluster --name <name> --query cluster.version`): 1.12
- AMI Version: built using packer from this repository at 88ca2ac5
- Kernel (e.g. `uname -a`):4.14.121-109.96.amzn2.x86_64
- Release information (run `cat /etc/eks/release` on a node):

```
BASE_AMI_ID="ami-0cb72367e98845d43"
BUILD_TIME="Mon Jun 10 05:17:18 UTC 2019"
BUILD_KERNEL="4.14.114-105.126.amzn2.x86_64"
ARCH="x86_64"
```

**Comment by whereisaaron** [comment]
@willthames perhaps you need to use a newer version that specifies `TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256` (with EC key)? See #279.

**Comment by willthames** [comment]
Confirmed fixed by #279.
